gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -915,10 +966,18 @@ gst_tensor_src_iio_set_property (GObject * object, guint prop_id,
       self->device.name = g_value_dup_string (value);
```

Comment:
What if ```self->device.name``` is NOT null?

Reply:
Handled.
Also added check for setting properties in null/ready state only.